app/views/meursing_lookup/steps/_end.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= back_link %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl">Look up a Meursing code</span>
+
+    <div class="govuk-panel govuk-panel--confirmation xgovuk-panel--semi-confirmation govuk-!-margin-top-3">
+      <h1 class="govuk-panel__title">
+        Your Meursing code
+      </h1>
+
+      <div class="govuk-panel__body">
+        The Meursing additional code for a product with this composition is <strong>7<%= current_step.meursing_code %></strong>.
+      </div>
+    </div>
+
+    <h2 class="govuk-heading-m">What next</h2>
+
+    <p class="govuk-body">
+      Use these four digits together with the ten-digit commodity code from Trade Tariff to work out duties applicable to certain complex agri-foods on the the Northern Ireland Tariff, when your import is considered to be 'at risk'.
+    </p>
+
+    <% if last_commodity_code.present? %>
```

Comment:
I am thinking with we call this 
`current_commodity_code` 🤔 

last_commodity_code sounds a DB querry